StringHelper.cs

```diff
@@ -60,46 +60,57 @@ public static int CountSentenses(string text)
         /// <summary>
         /// function for counting letters
         /// </summary>
-        public static int Countwords(string text)
+
+        public static int CountVowels(string text)
+        {
+            char[] chars = text.ToCharArray();
+            int numberOfVowes = 0;
+
+            for (int i = 0; i < text.Length; i++)
+            {
+                if (char.IsLetter(chars[i]) == true &&( chars[i].ToString().ToLower() == "a" || chars[i].ToString().ToLower() == "e" || chars[i].ToString().ToLower() == "i" || chars[i].ToString().ToLower() == "o" || chars[i].ToString().ToLower() == "u" || chars[i].ToString().ToLower() == "y"))
+                {
+                    numberOfVowes = numberOfVowes + 1;
+                }
+            }
+            return numberOfVowes;
+        }
+
+        public static int CountConstatnts(string text)
+
```

Comment:
it is good practice not to have new line between function name and brackets. please remove if you can